### DISCUSSION TITLE
API で 記事に対する Stock の数が簡単に取得できるようにして欲しい

### DISCUSSION THREAD

**Discussion by ekzemplaro** [increments/qiita-discussions#59] [要望・アイデア・不具合]
Contribution のカウントの方法が LGTM だけでなく、Stock も考慮されるようになりましたが、API がそれに対応していません。記事に対して、LGTM が取れるように、Stock も取れるような API があればと思います。

**Comment by ohakutsu** [comment]
@ekzemplaro 
ご意見ありがとうございます！
現在進行中のプロジェクトのスケジュールの兼ね合いで、申し訳ないのですがすぐに対応は難しいと考えています

代替策としてすでにご存知かもしれませんが、以下の方法でストック数を取ることができます

1. https://qiita.com/api/v2/docs#get-apiv2itemsitem_idstockers で記事をストックしているユーザーを取得
2. ユーザー数をカウントする

我々からしても不便であると感じておりますが、こちらの方法でお願いします 🙇

**Comment by wataru86** [comment]
@ekzemplaro
大変お待たせいたしました :bow: 記事のストックされた数をAPIで取得できるようになりました！
仕様は以下ドキュメントからご確認ください。
https://qiita.com/api/v2/docs#%E6%8A%95%E7%A8%BF

**Reply by ekzemplaro** [reply]
ありがとうございます。
さっそく、Stocks を表示するようにしました。
次のページをご覧下さい。
https://ekzemplaro.org/qiita/
